### ISSUE TITLE
cursor: pointer missing on Passed Audits

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/lighthouse#5958]
In the report, it's clickable but we dont have `cursor` set on it. :/  should be fixed.

Probably applies to non-applicable and manual audit groups too.

**Comment by midzer** [comment]
Hi again :)
After playing around a little bit, I found a possible solution for overal convenient cursor behaviour:
For
```
.lh-audit__header,
.lh-audit-group--manual,
.lh-passed-audits,
.lh-audit-group--not-applicable
```
we could set `cursor: pointer`. Miserably this overrides their children and I think we don't want a pointer on non-clickable text. So we have to restore initial value for:
```
lh-audit-group__description,
lh-audit__description,
lh-details
```
Looking forward to start PR if you want to have a look on this one.

Have a nice week
midzer

**Comment by paulirish** [comment]
fixed by #5977
